portfolio/src/main/webapp/script.js

```diff
@@ -46,22 +46,24 @@ function createMap() {
  * Draws a chart and adds it to the page.
```

Comment:
Can you describe what kind of data and what kind of chart?

Reply:
Done!